### PR TITLE
Add tests for calling associate/dissociate via the HRC proxy contract

### PR DESCRIPTION
**Pull request by lukelee-sl**
**Description**:
Test that the `associate()` and `dissociate()` calls can be made via the HRC interface which calls the proxy facade contract in services

Fixes #

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by georgi-l95 — APPROVED** [review]
LG

File: test/hts-precompile/hrc/HRCContract.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    before(async function () {
+        signers = await ethers.getSigners();
+        tokenCreateContract = await utils.deployTokenCreateContract();
+
+        // This contract is a wrapper for the associate() and dissociate() functions
+        hrcContract = await utils.deployHRCContract();
+        tokenAddress = await utils.createFungibleToken(tokenCreateContract, signers[0].address);
+
+        // create an interface for calling functions via redirectForToken()
+        IHRC = new ethers.utils.Interface((await hre.artifacts.readArtifact("IHRC")).abi);
```
Comment (natanasow) [inline test/hts-precompile/hrc/HRCContract.js:52]: nit: let's define `IHRC` as local variable under `describe` scope

**Review by natanasow — DISMISSED** [review]
LGTM

**Review by Nana-EC — DISMISSED** [review]

**Review by georgi-l95 — APPROVED** [review]
👍

**Review by Nana-EC — APPROVED** [review]
LGTM

**Pull request merged**